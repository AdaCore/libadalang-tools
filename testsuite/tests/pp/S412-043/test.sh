gnatpp -q --pipe cvt_example.ada
