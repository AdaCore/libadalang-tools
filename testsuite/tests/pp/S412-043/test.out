
Info: AdaCore provides a new formatter GNATformat, currently in beta. This will supersede GNATpp when leaving the beta program.
-----

function Cvt is new Unchecked_Conversion
  (Source => Integer, Target => Tds_Unsigned_T);
-- Add Cvt for Tds_Integer_T if it is ever different from Integer.
